cohort_4/playbook/ACTIVITIES.md

```diff
@@ -689,6 +689,35 @@ Uniqueness is also important - no two people should be paired together again
 unless everyone has already been paired with each other. This will help members
 of the club all form bonds with each other.
 
+## Types of Pairing
+
+### Pair programming between beginner-advanced pairs
+
+Pair programming between beginner and advanced hackers is fairly similar to
+normal pair programming. They should still be rotating regularly. The advanced
+hacker will have a tendnacy to drive the coding completely and club leader may
+have to help them resist that urge.
```

Comment:
Perhaps make this a bit less declarative?

Ex. "Do note: we've observed that advanced hackers often have the tendency to want to drive the coding completely. You may need to have to help them resist that urge by doing x, y, and z."